src/NuGet.Clients/NuGet.PackageManagement.PowerShellCmdlets/NuGetPowerShellBaseCommand.cs

```diff
@@ -248,17 +259,27 @@ protected async Task CheckMissingPackagesAsync()
 
                     using (var cacheContext = new SourceCacheContext())
                     {
+                        var logger = new LoggerAdapter(this);
+
+                        var signedPackageVerifier = new PackageSignatureVerifier(SignatureVerificationProviderFactory.GetSignatureVerificationProviders());
+
                         var downloadContext = new PackageDownloadContext(cacheContext)
                         {
-                            ParentId = OperationId
+                            ParentId = OperationId,
```

Comment:
Same as above, I don't follow how you understand that the package save mode is v3.

Reply:
Discussed offline. 

ProjectContext will always be V2, DownloadContext (used only in the gpf case will always be V3)